### PR TITLE
Added option to privide a watch directory

### LINKED ISSUE #2: Add an option for which folder to watch
Could you add an option to define which folder to watch? The whole app folder is a bit of an overkill and a bit useless to have bless run when I change a php file.

### PR DESCRIPTION
**Pull request by junkystu**
Fixes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,8 @@ require('laravel-elixir-bless');
 
 elixir(function(mix) {
     mix.bless('app.css', {
-        imports: false
+        imports: false,
+        watch: './resources/less/*.less'
```
Comment (jbrooksuk) [inline README.md]: Less only?

Comment (junkystu) [inline README.md]: That is only an example of a pattern that can be provided via the option. Elixir's `watch()` is internally using [gulp watch](https://www.npmjs.com/package/gulp-watch) so any pattern accepted by that can be provided (e.g. a directory, a file, file patterns, etc).

**Comment by junkystu** [comment]
I've updated the readme to make it more clear what the option is for.

**Comment by jbrooksuk** [comment]
Perfect, thanks!

**Pull request merged**